modules/table/src/main/java/org/apache/ignite/internal/table/TableView.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.table;
+
+import org.apache.ignite.internal.schema.ColumnsExtractor;
+import org.apache.ignite.internal.schema.SchemaRegistry;
+import org.apache.ignite.internal.storage.index.StorageHashIndexDescriptor;
+import org.apache.ignite.internal.storage.index.StorageSortedIndexDescriptor;
+import org.apache.ignite.internal.table.distributed.PartitionSet;
+import org.apache.ignite.network.ClusterNode;
+import org.apache.ignite.table.Table;
+import org.apache.ignite.table.Tuple;
+import org.apache.ignite.table.mapper.Mapper;
+
+/** Internal table view interface. */
+public interface TableView extends Table {
+    /**
+     * Gets a table id.
+     *
+     * @return Table id as UUID.
+     */
+    int tableId();
+
+    /**
+     * Provides current table with notion of a primary index.
+     *
+     * @param pkId An identifier of a primary index.
+     */
+    void pkId(int pkId);
+
+    /** Returns an identifier of a primary index. */
+    int pkId();
+
+    /** Returns an internal table instance this view represents. */
+    InternalTable internalTable();
+
+    /**
+     * Gets a schema view for the table.
+     *
+     * @return Schema view.
```

Comment:
Why does this comment style (it has return section) differ from `internalTable()` and `pkId()`? Can you use the same style for all javadoc here?